projects/policekillings/db/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mysql:5.5
+
+ENV MYSQL_DATABASE milehigh_grassroots_law
+ENV MYSQL_ROOT_PASSWORD glp411!
+COPY dump.sql /docker-entrypoint-initdb.d/
```

Comment:
Let's not commit these values since they are sensitive; I was able to see this PR while not logged in, so we should change the password right away to avoid any issues with this database moving forward.

If you need to access sensitive data in your application, you can do a few things. For this use case, I'd recommend just adding this file to the list of files in `.gitignore`, so it will be available to you locally but won't be available included in committed files.

Reply:
Ah ok! I did not know that. I just learned how to do this by following https://www.youtube.com/watch?v=_vejzukmn4s , and pushed a new version of this branch to resolve.